apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/NoOrdersContent.tsx

```diff
@@ -1,16 +1,60 @@
 import { ReactNode } from 'react'
 
-import cowMeditatingV2 from '@cowprotocol/assets/cow-swap/meditating-cow-v2.svg'
+import surprisedCowAnimationDark from '@cowprotocol/assets/lottie/surprised-cow-dark.json'
```

Comment:
Could you lazy-load the jsons please?
Their size is 300kb in total with is significant, I don't think we can afford to increase the bundle size so much.

Reply:
Addressed this now.